fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -160,6 +161,8 @@
 
     private static final PropertyConverter propertyConverter = new PropertyConverter();
 
+    public static final String LDPCV_TIME_MAP = "TimeMap";
```

Comment:
Is there a difference between a FEDORA_TIME_MAP and this LDPCV_TIME_MAP?  Do we need to definitions of this?  They should be one in the same...??

Reply:
Actually, Never mind. I see what's going on.